### PR TITLE
Update tagesschau.de

### PR DESCRIPTION
**Pull request by DavidLiedke**
PR #13790 not mantained anymore. 

Remove Test Url for tagesschau.de in PR #14660

### CODE REVIEW

File: src/chrome/content/rules/tagesschau.de.xml [hunk 1]
```diff
@@ -1,52 +1,43 @@
...
+	<target host="meta.tagesschau.de"/>
+	<target host="prodforum.tagesschau.de"/>
+	<target host="service.tagesschau.de"/>
+	<target host="wetter.tagesschau.de"/>
+	<target host="programm.tagesschau24.de"/>
 
 	<securecookie host="^www\.tagesschau\.de$" name=".+" />
 
-	<rule from="^http://tagesschau\.de/"
-		to="https://www.tagesschau.de/"/>
```
Comment (J0WI) [inline src/chrome/content/rules/tagesschau.de.xml]: Please keep this rule and add a comment on top for the issue on `^`.

File: src/chrome/content/rules/tagesschau.de.xml [hunk 2]
```diff
@@ -1,52 +1,43 @@
...
-		- Bugs, on:
-
-			- wetter and www from de.sitestat.com *
-			- wetter and www from gsea.ivwbox.de *
-			- wetter and www from tagessch.ivwbox.de *
-			- www from ts.flyp.tv *
-
-	* Secured by us
+		karten.tagesschau.de
+		wahl.tagesschau.de
```
Comment (J0WI) [inline src/chrome/content/rules/tagesschau.de.xml]: I was not able to reproduce mixed content issues on those hosts.

File: src/chrome/content/rules/tagesschau.de.xml [hunk 3]
```diff
@@ -1,4 +1,6 @@
 <!--
+	tageschau.de redirect to www version
```
Comment (J0WI) [inline src/chrome/content/rules/tagesschau.de.xml]: The issue is, that https is refused on `^`

File: src/chrome/content/rules/tagesschau.de.xml (hunk 2 above)
Comment (DavidLiedke) [reply src/chrome/content/rules/tagesschau.de.xml]: My Firefox and Google Chrome give me this feedback with the mixed content.
Or see here:
wahl.tagesschau.de https://www.whynopadlock.com/results/9cf16086-e537-4f47-a650-df357f9fbc8b
karten.tagesschau.de https://www.whynopadlock.com/results/5efaad71-cf13-4b5f-9272-d8318090019d

Comment (J0WI) [reply src/chrome/content/rules/tagesschau.de.xml]: These are trackers, you can safely ignore them.

File: src/chrome/content/rules/tagesschau.de.xml [hunk 4]
```diff
@@ -1,52 +1,52 @@
...
+	<target host="wetter.tagesschau.de"/>
+	<target host="programm.tagesschau24.de"/>
 
 	<securecookie host="^www\.tagesschau\.de$" name=".+" />
 
-	<rule from="^http://tagesschau\.de/"
+    <rule from="^http://tagesschau\.de/"
 		to="https://www.tagesschau.de/"/>
 
 		<test url="http://www.tagesschau.de/" />
```
Comment (J0WI) [inline src/chrome/content/rules/tagesschau.de.xml]: The test url can be removed, but you have to add `tagesschau.de` as target.

File: src/chrome/content/rules/tagesschau.de.xml [hunk 5]
```diff
@@ -1,52 +1,53 @@
...
+	<target host="prodforum.tagesschau.de"/>
+	<target host="service.tagesschau.de"/>
+	<target host="wahl.tagesschau.de"/>
+	<target host="wetter.tagesschau.de"/>
+	<target host="programm.tagesschau24.de"/>
 
 	<securecookie host="^www\.tagesschau\.de$" name=".+" />
 
-	<rule from="^http://tagesschau\.de/"
+    <rule from="^http://tagesschau\.de/"
```
Comment (J0WI) [inline src/chrome/content/rules/tagesschau.de.xml]: Please convert spaces to tabs and remove trailing tabs (around `Invalid Certificate-Chain` block)

**Comment by DavidLiedke** [comment]
Done.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**